### PR TITLE
Remove unnecessary sandbox-status related code

### PR DESCRIPTION
**Pull request by vilmakoo**
Add test for SandboxPosterJob
Remove unnecessary times_sent_to_sandbox from exercises.

 :open_mouth:  :sweat_drops: :raised_hands: :metal:

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -8,8 +8,7 @@ def index
   end
 
   def create
-    token = oauth_client.password.get_token(params[:login], params[:password])
-    session[:oauth_token] = token.token
+    set_token
```
Comment (pirtnea) [inline app/controllers/sessions_controller.rb]: Good code  :raised_hands:

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -31,6 +30,11 @@ def destroy
 
   private
 
+  def set_token
```
Comment (nygrenh) [inline app/controllers/sessions_controller.rb]: Maybe `fetch_token` since it makes a network request?

File: spec/jobs/sandbox_poster_job_spec.rb [hunk 3]
```diff
@@ -2,6 +2,21 @@
 
 require 'rails_helper'
 require 'tarballer'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
```
Comment (pirtnea) [inline spec/jobs/sandbox_poster_job_spec.rb]: Fake :open_mouth:

File: db/migrate/20170824083021_add_times_sent_to_sandbox_to_exercises.rb [hunk 4]
```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-class AddTimesSentToSandboxToExercises < ActiveRecord::Migration[5.1]
```
Comment (pirtnea) [inline db/migrate/20170824083021_add_times_sent_to_sandbox_to_exercises.rb]: Whoops

File: db/migrate/20170829113355_remove_times_sent_to_sandbox_from_exercises.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class RemoveTimesSentToSandboxFromExercises < ActiveRecord::Migration[5.1]
```
[the remark this answers is not in the document]
Comment (vilmakoo) [reply db/migrate/20170829113355_remove_times_sent_to_sandbox_from_exercises.rb]: :fu:

[the remark this answers is not in the document]
Comment (vilmakoo) [reply db/migrate/20170829113355_remove_times_sent_to_sandbox_from_exercises.rb]: Sorry Hound.

**Pull request merged**